development/mimir-ingest-storage/.data-minio/.gitignore

```diff
@@ -0,0 +1,5 @@
+*
+!mimir-ruler
```

Comment:
I think we should have the same set of ignores for `mimir-ingest` bucket too.

Reply:
there's no mimir-ingest bucket because we're using kafka.

I also didn't have luck persisting the kafka data. After `docker-compose down -v` the cluster ID seems to be in conflict with the one on disk. At this point the volume is just some docker volume. I don't think that's that bad